tests/rptest/services/utils.py

```diff
@@ -178,11 +201,16 @@ def __init__(self, test_context, allow_list, logger,
         # Prepare capture functions
         self.kubectl = kubectl
 
+    @staticmethod
+    def _get_badlines_exception(bad_loglines):
+        # For Cloud deployments use corresponding Exception Class
+        return BadLogLinesCloud(bad_loglines)
```

Comment:
Why is the exception type different?

Reply:
It is child from original one that is handling CloudBroker passed to it properly

Reply:
Thanks, I see it's to allow `_get_hostname` to be dynamic.